npm install failing

`$ npm install ciscospark` seems to be failing for myself and a colleague, giving: 

```
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.3.1/bin/node" "/usr/local/bin/npm" "install" "--save" "ciscospark"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5

npm ERR! No compatible version found: @ciscospark/spark-core@^0.7.0-alpha.f22f099e
npm ERR! Valid install targets:
npm ERR! 0.7.0-alpha.ce9393d4, 0.7.0-alpha.a5bc0482, 0.7.0-alpha.3266717c, 0.6.9, 0.6.6, 0.6.4, 0.6.3, 0.6.1, 0.6.0, 0.6.0-35, 0.6.0-34
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rilogan/Projects/SparkVideo/npm-debug.log
```

`$ npm view @ciscospark/plugin-phone versions` doesn't seem to show the asked for *0.7.0-alpha.f22f099e* version. 

For the record, `$ npm install ciscospark@0.6.9` works fine.

Hi @ianwremmel, when I try and follow the build instructions (in order to build for the browser functionality) `$ npm install` gives `npm ERR! 404  'spark-js-sdk--test-users' is not in the npm registry`. Removing that dependency allows install to complete, but then `npm run bootstrap` fails also. 

It looks like swapping out `$ npm run bootstrap` for just `$ lerna bootstrap` allows the rest of the build to run properly and generate the bundle. This was working off of the c49ca68 commit. 

Please let me know if it's just me doing something wrong! I don't have much experience with Node/npm.

--- a/tooling/circle-scripts/publish.sh
+++ b/tooling/circle-scripts/publish.sh
@@ -51,5 +51,5 @@ echo "Tricking npm website into updating the README"
 # See https://github.com/npm/newww/issues/389#issuecomment-188428605 and
 # https://github.com/lerna/lerna/issues/64 for details
 npm run lerna -- exec --scope ciscospark -- npm version --no-git-tag-version "${NEW_VERSION}-readmehack"
-npm run lerna -- exec --scope ciscospark -- npm publish
+npm run lerna -- exec --scope ciscospark -- npm publish --tag=readmehack
 npm run lerna -- exec --scope ciscospark -- npm unpublish ciscospark@"${NEW_VERSION}-readmehack"

--- a/tooling/jenkins-scripts/gate-jenkins.sh
+++ b/tooling/jenkins-scripts/gate-jenkins.sh
@@ -13,6 +13,15 @@ export XUNIT=true
 LOG_FILE="$(pwd)/test.log"
 rm -f "${LOG_FILE}"
 
+PWD=`pwd`
+
+cd /tmp
+ls lerna > /dev/null 2> /dev/null || git clone git@github.com:ianwremmel/lerna.git:
+git checkout de0f7f2
+npm link
+cd pwd
+npm link lerna
+
 # INSTALL
 echo "Installing modular SDK dependencies"
 npm run bootstrap

